app/policies/invite_policy.rb

```diff
@@ -6,7 +6,11 @@ def index?
   end
 
   def create?
-    role.can?(:invite_users)
+    role.can?(:invite_users) && (current_user.created_at <= (Time.now.utc - 7.days) || unrestricted?)
```

Comment:
It would make sense for `7.days` to be a constant, if not configurable. Assuming such a constant would be called `MIN_TIME_BEFORE_INVITE`, a slightly more idiomatic way to write it would be `current_user.created_at <= MIN_TIME_BEFORE_INVITE.ago`

Reply:
Should that constant be in this file, or added to the validator?